.github/workflows/docs-build.yml

```diff
@@ -85,86 +82,40 @@ jobs:
           mkdir -p $PYPI_CACHE_DIR  # in case cache was not hit
           ls -lh $PYPI_CACHE_DIR
           mkdir -p pypi_pkgs  # in case template is not pulled
-          pip install .[extra,cloud,ui] -U -r requirements/${{ matrix.pkg-name }}/docs.txt \
+          pip install .[all] -U -r requirements/${{ matrix.pkg-name }}/docs.txt \
```

Comment:
This will include testing and example deps. I suggest reverting this change

Reply:
I think it is fine...

Reply:
@Borda what does "all" include in addition to the previous ones?

Reply:
Where can I find the mapping from these tags to dependencies?

Reply:
then all `*-all` are aggregated as `all`